chapters/ch06/3.spell-correction.ipynb

```diff
@@ -578,7 +579,7 @@
        "  <thead>\n",
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
-       "      <th>misspell</th>\n",
+       "      <th>misspelling</th>\n",
```

Comment:
The committed notebook before was clearly wrong. The code is supposed to output "misspelling", but the output for this column was "misspell", which I'm assuming means the notebook likely wasn't rerun after the code change was made.

I've updated the tables in the manuscript to say "misspelling" for the column header to match the code, assuming the code is correct.

Reply:
The code is correct.